Library/Homebrew/dev-cmd/test-bot.rb

```diff
@@ -658,15 +658,17 @@ def homebrew
       return if @skip_homebrew
 
       if @tap.nil?
-        tests_args = []
+        tests_args_classic = []
+        tests_args_generic = %w[--generic --only=integration_cmds]
+        tests_args_coverage = %w[--no-compat]
         if RUBY_TWO
-          tests_args << "--official-cmd-taps"
-          tests_args << "--coverage" if ENV["TRAVIS"]
+          tests_args_classic << "--official-cmd-taps"
+          tests_args_generic << "--official-cmd-taps"
+          tests_args_coverage << "--coverage" if ENV["TRAVIS"]
         end
-        test "brew", "tests", *tests_args
-        test "brew", "tests", "--generic", "--only=integration_cmds",
-                              *tests_args
-        test "brew", "tests", "--no-compat"
+        test "brew", "tests", *tests_args_classic
+        test "brew", "tests", *tests_args_generic
+        test "brew", "tests", *tests_args_coverage
```

Comment:
Fair point. I was personally more concerned with line length and with a more uniform look (and making sure that all invocations have individual variables for their arguments to avoid false sharing of arguments).